pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementsLQubit.hpp

```diff
@@ -271,7 +271,7 @@ class Measurements final
     /**
      * @brief Expectation value for a Observable with shots
      *
-     * @param ob Observable
+     * @param obs Observable
      * @param shots Vector of shot number to measurement
```

Comment:
```suggestion
     * @param shot_range Vector of shot number to measurement
```
`num_shots` may be missing too.

Reply:
Nice catch! Done